### PR TITLE
Test support for Adaptive Cards

### PR DESCRIPTION
**Pull request by photodesignch**
Adding adaptive cards support

### CODE REVIEW

File: test/commands_map.ts [hunk 1]
```diff
@@ -193,6 +195,23 @@ var commands_map: CommandValuesMap = {
             sendActivity(res, server_content.suggested_actions_card);
         }
     },
+    // CHRIS : TODO : taking out
```
Comment (photodesignch) [inline test/commands_map.ts]: forgot to remove these comments before I commit. I will remove them.

**Comment by danmarshall** [comment]
Hi Chris, can you also rename the PR ?

**Comment by danmarshall** [comment]
Perhaps it should be names "Adding Adaptive Cards tests."

File: test/mock_dl/index.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import bodyParser = require('body-parser');
 import * as path from 'path';
 
 const app = express();
+const fs = require('fs');
```
Comment (eanders-MS) [inline test/mock_dl/index.ts]: Preferred to import like this: `import * as fs from 'fs';`

File: test/commands_map.ts [hunk 3]
```diff
@@ -4,13 +4,15 @@ import * as Nightmare from 'nightmare';
 import * as express from 'express';
 declare let module: any;
 
+interface jsonObj {}
```
Comment (eanders-MS) [inline test/commands_map.ts]: This seems unnecessary.

Comment (photodesignch) [inline test/commands_map.ts]: removed

File: test/mock_dl/index.ts (hunk 2 above)
Comment (photodesignch) [inline test/mock_dl/index.ts]: changed to import * as fs from 'fs'

Comment (photodesignch) [inline test/mock_dl/index.ts]: done

File: test/cards/AdaptiveCardStyles.xaml [hunk 4]
```diff
@@ -0,0 +1,305 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```
Comment (danmarshall) [inline test/cards/AdaptiveCardStyles.xaml]: can you remove this file entirely - unless there is some reason we need it.

Comment (photodesignch) [inline test/cards/AdaptiveCardStyles.xaml]: @dandriscoll Will consolidate this folder to remove unnecessary files. Currently, this is just showing I copied all the files from AdaptiveCards folder.

**Comment by reyesrico** [comment]
I suggest to add only the 5 cards that we are using in Channel Inspector instead of the 65 files. Lets use: Weather, BingSports, Simple, FlightUpdate and CalendarReminder.

**Pull request closed without merging**